test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageSender.kt

```diff
@@ -15,49 +15,90 @@
  */
 package com.epam.drill.test2code.coverage
 
+import com.epam.drill.plugins.test2code.common.api.*
+import com.github.luben.zstd.*
 import kotlinx.coroutines.*
-import mu.KotlinLogging
-import java.util.concurrent.Executors
-import kotlin.coroutines.CoroutineContext
+import kotlinx.serialization.protobuf.*
+import mu.*
+import java.util.*
+import java.util.concurrent.*
+
+private const val RETENTION_QUEUE_LIMIT = 2000L
```

Comment:
Naming should indicate units - add suffix `_BYTES`, `_MB`, etc

Reply:
added